openhtf/core/test_state.py

```diff
@@ -251,16 +250,26 @@ def finalize_normally(self):
     if self._is_aborted():
       return
 
-    allowed_outcomes = {measurements.Outcome.PASS}
-    if conf.allow_unset_measurements:
-      allowed_outcomes.add(measurements.Outcome.UNSET)
+    all_skipped = bool(self.test_record.phases)
```

Comment:
add comments here or at least something in `docs/` that explains what all this does